src/client/swagger/unity.yml

```diff
@@ -839,6 +839,13 @@ components:
         id:
           type: string
           description: account id
+        quartzId:
+          type: string
+          description: account id in quartz
+        onboardingState:
+          type: string
+          # TODO(ariel): get clarity on this
+          description: stage of onboarding the account is in
```

Comment:
These two properties seem to only be relevant for operators visiting the `AccountView` page. That is to say, these fields are only needed when an operator is viewing an account-specific page